Author: Jamie Strandboge <jamie@canonical.com>
Description: newer lxc uses 'silent' when remounting on shutdown. Silence that
 denial too

Index: lxc-1.1.0~alpha1/config/apparmor/abstractions/container-base
===================================================================
--- lxc-1.1.0~alpha1.orig/config/apparmor/abstractions/container-base
+++ lxc-1.1.0~alpha1/config/apparmor/abstractions/container-base
@@ -11,6 +11,7 @@
 
   # ignore DENIED message on / remount
   deny mount options=(ro, remount) -> /,
+  deny mount options=(ro, remount, silent) -> /,
 
   # allow tmpfs mounts everywhere
   mount fstype=tmpfs,
